docs/api-v1.json

```diff
@@ -0,0 +1,1136 @@
+            "description": "error message",
+            "example": "used a teapot instead of a coffeepot"
+          },
+          "path": {
+            "type": "string",
+            "description": "path of the request that caused the error",
+            "example": "/v1/coffee/"
+          },
+          "status": {
+            "type": "integer",
+            "description": "status code",
+            "example": 4418
+          },
+          "timestamp": {
+            "$ref": "#/components/schemas/Date"
+          }
+        }
+      },
+      "UserGet": {
+        "description": "User object with full information",
+        "type": "object",
+        "readOnly": true,
+        "properties": {
+          "id": {
+            "$ref": "#/components/schemas/UUID"
+          },
+          "creation_date": {
+            "$ref": "#/components/schemas/Date"
+          },
+          "authentication": {
+            "$ref": "#/components/schemas/AuthenticationData"
+          },
+          "role": {
+            "$ref": "#/components/schemas/Role"
+          },
+          "details": {
+            "$ref": "#/components/schemas/UserDetails"
+          }
+        }
+      },
+      "UserGetPublic": {
+        "description": "User object with reduced information",
+        "type": "object",
+        "readOnly": true,
+        "properties": {
+          "id": {
+            "$ref": "#/components/schemas/UUID"
+          },
+          "authentication": {
+            "description": "Authentication data object",
+            "type": "object",
+            "properties": {
+              "email": {
+                "type": "string",
+                "description": "email of the user",
+                "example": "erika.mustermann@uzh.ch"
+              }
+            }
+          },
+          "details": {
+            "description": "User details object",
+            "type": "object",
+            "properties": {
+              "first_name": {
+                "type": "string",
+                "description": "first name of the user",
+                "example": "Erika"
+              },
+              "last_name": {
+                "type": "string",
+                "description": "last name of the user",
+                "example": "Mustermann"
+              },
+              "biography": {
+                "type": "string",
+                "description": "biography of the user",
+                "example": "I am a very nice person"
+              }
+            }
+          }
+        }
+      },
+      "UserPostPut": {
+        "description": "User object used to create or update a user",
+        "type": "object",
+        "properties": {
+          "authentication": {
+            "$ref": "#/components/schemas/AuthenticationData"
+          },
+          "role": {
+            "$ref": "#/components/schemas/Role"
+          },
+          "details": {
+            "$ref": "#/components/schemas/UserDetails"
+          }
```

Comment:
shouldn't it also include the publisher?

Reply:
the get sends a public user version `private UserGetPublicDto publisher;`

Reply:
yes , the owner of the listing should also be added, no?

Reply:
but this is UserPostPut and has nothing to do with listing. I think I just don't get it :(

Reply:
oh yes, you're absolutely right. My bad all good!